### DISCUSSION TITLE
ch1/ch1-08

### DISCUSSION THREAD

**Discussion by giscus** [gopl-zh/gopl-zh.github.com#208] [General]
# ch1/ch1-08

<The Go Programming Language>中文版

https://golang-china.github.io/gopl-zh/ch1/ch1-08.html

**Comment by jiechen257** [comment]
推荐类似 copilot 的 ai 工具 ——  codeium（免费！

**Comment by lin-snow** [comment]
第一章完结撒花🎉！

写的很不错！
